### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lirad**
Completed project first submission

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,244 @@
...
+                    <div class="actions-buttons">
+                        <div class="actions-buttons-wrapper">
+                            <div>
+                                <i class="fas fa-plus"></i>
+                                add to
+                            </div>
+                            <div>
+                                <i class="fas fa-plus"> </i>
+                                share
+                            </div>
```
Comment (peterrobert) [inline index.html]: this should be a button as specified at the requirements
share

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO :wave: @lirad AND @enkog  AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT.

### THIS IS QUITE AN IMPRESSIVE WORK FOR A FIRST REVIEW AND FIRST PROJECT :clap: 
However, before continue you need to make the following changes.

- [x] Your readme is good but let make it even better. You need to make your readme more descriptive. Kindly take a step back and add all the required sections in a readme file.

- [x] Your video is well embedded but the height is quite small compared to the original kindly improve on that by increasing the height
embed

### kindly make those changes and submit your project for another review.

### HAPPY CODING :computer:

File: index.html (hunk 1 above)
Comment (enkog) [reply index.html]: We did the corrections. please can you validate it.

**Comment by lirad** [comment]
Corrections asked were made:

- Improved readme
- Corrected video height
- Transformed "share" in button

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Webbrowser and access to internet
+
+### Install
+
+There is no need to installation, just run the index.html file
+
```
Comment (kitpao) [inline README.md]: - [ ] **Mandatory:** The githack raw you included made your project look nice ✔, however, what my colleague meant with descriptive readme file can be found in this **installation section**, where you have to put the instructions to download your repo, such as, the steps in the terminal to download your project ( `git clone <the link to your repo>` ) and the steps to open a live version in localhost (clicking in the downloaded index.html file).

File: index.html [hunk 3]
```diff
@@ -1,11 +1,244 @@
...
+
+                        <div class="chanel-name-sub-wrapper">
+                            <div class="channel-name">
+                                Microverse
+                            </div>
+                            <div class="subscribe-button">
+                                <span class="red-part">
+                                    <i class="fab fa-youtube"></i>
+                                    Subscribe
+                                </span>
```
Comment (kitpao) [inline index.html:70]: - [ ] **[Optional]** You named it subscribe-button but you didn't use a button tag image

File: index.html [hunk 4]
```diff
@@ -1,11 +1,244 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
```
Comment (kitpao) [inline index.html:5]: **[Optional]** 🍀 Friendly note, in your project you used 4 spaces in your document, in future projects you will be required to use a 2 space indentation to follow html conventions

**Review by kitpao — CHANGES_REQUESTED** [review]
## 👋 Hello @lirad and @enkog,

I've reviewed your project and I have some comments about your project:
✔ You made a nice use of the containers

✔ You fixed nicely the height of the video

🔍Side note: As a reminder, the branch name can be anything that gives the idea of the things you are doing, in GitHub flow it doesn't have to be "development" related

And you only have **_1 mandatory change_** in the README file, kindly take a look below to fix it: ⬇️

### ♻️ Status: Your project requires some additional changes ♻️

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao
##### 👉 You can find me in your completition form as Kitzia Paola Vidal Marroquin

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by lirad** [comment]
Complied with all suggestions made by the TSE:
1. Updated readme file;
2. Transformed subscribe in button tag;
3. Adjusted ident from index.html

**Review by expjazz — APPROVED** [review]
## Great work Team! The project is approved! :clap: 

#### As a reminder, for the next pull requests for the next projects please use a more meaningful title and an informative description of the project. You did well on the changes required by my colleagues.

### Welcome to Microverse :handshake: 

Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Pull request merged**